libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/MapboxNavigationView.kt

```diff
@@ -0,0 +1,73 @@
+package com.mapbox.navigation.dropin
+
+import com.mapbox.maps.MapView
+import com.mapbox.navigation.core.arrival.ArrivalObserver
+import com.mapbox.navigation.core.directions.session.RoutesObserver
+import com.mapbox.navigation.core.trip.session.BannerInstructionsObserver
+import com.mapbox.navigation.core.trip.session.LocationObserver
+import com.mapbox.navigation.core.trip.session.RouteProgressObserver
+import com.mapbox.navigation.core.trip.session.TripSessionStateObserver
+
+interface MapboxNavigationView {
+    val navigationView: NavigationView
```

Comment:
I can't mark it private since it's in an interface. Perhaps it's not the most elegant but it's the same reference the developer already has. It doesn't cause any problems.

Reply:
My question is why do we even need it? Can't we just remove it?

Reply:
The interface needs a reference to NavigationView.